docs/api-reference.md

```diff
@@ -29,6 +29,7 @@ apiVersion: package-operator.run/v1alpha1
 kind: ClusterObjectDeployment
 metadata:
   name: example
+  namespace: default
```

Comment:
this file is not part of the PR

Reply:
fix: rebase onto main and redo the commit, k8s-docgen has been fixed